src/test/java/org/fundacionjala/coding/manuel/Kata8Test.java

```diff
@@ -0,0 +1,131 @@
+package org.fundacionjala.coding.manuel;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+/**
+ * Created by Juan Manuel on 3/11/2018.
+ */
+public class Kata8Test {
+    private Kata8 kata8;
+    private static final String COMMON_LINE1 = " _  _  _  _  _  _  _  _  _ ";
+    private static final String COMMON_LINE2 = "|_||_||_||_||_||_||_||_||_|";
+    private static final String COMMON_LINE3 = "  |  |  |  |  |  |  |  |  |";
+    private static final String COMMON_LINE4 = " _| _| _| _| _| _| _| _| _|";
+    /**
+     * Method to initialize.
+     */
+    @Before
+    public void initialize() {
+        kata8 = new Kata8();
+    }
+
+    /**
+     * First test.
+     */
+    @Test
+    public void test1BankKata8() {
+        String secondRow = "| || || || || || || || || |";
+        assertEquals("000000000", kata8.bank(COMMON_LINE1, secondRow, COMMON_LINE2));
+    }
+
+    /**
+     * Second test.
+     */
+    @Test
+    public void test2BankKata8() {
+        String firstRow =  "                           ";
+        assertEquals("111111111 ERR", kata8.bank(firstRow, COMMON_LINE3, COMMON_LINE3));
+    }
+
+    /**
+     * Third test.
+     */
+    @Test
+    public void test3BankKata8() {
+        String thirdRow =  "|_ |_ |_ |_ |_ |_ |_ |_ |_ ";
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "|_ |_ |_ |_ |_ |_ |_ |_ |_ " 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)